src/lib.rs

```diff
@@ -112,11 +112,7 @@ pub struct Person {
     pub identifiers: pb::Identifiers,
     pub demographics: pb::Demographics,
     pub employment: pb::Employment,
-    pub bmi: BMI,
-    pub has_cardiovascular_disease: bool,
-    pub has_diabetes: bool,
-    pub has_high_blood_pressure: bool,
-
+    pub health: pb::Health,
```

Comment:
Here's the old Rust fields for health. Since the protobuf groups them into a nice `Health` message, we can just use that directly instead.